CHANGELOG.rst

```diff
@@ -1,40 +1,49 @@
 2.9.0.dev
 =========
 
-New Features
-------------
+**New Features**
```

Comment:
I also would prefer subtitles... Hmm perhaps [this](http://stackoverflow.com/questions/5464627/how-to-have-same-text-in-two-links-with-restructured-text) can work. I will try later.